scripts/process.py

```diff
@@ -1,33 +1,85 @@
 import os
-import urllib
 import csv
+import json
+import xlrd
+import requests
+
+from datetime import datetime, timedelta
 
 cache = 'cache'
-if not os.path.exists(cache):
-    os.makedirs(cache)
+data = '../data/gdp.csv'
+url = 'https://api.worldbank.org/v2/en/indicator/NY.GDP.MKTP.CD?downloadformat=excel'
```

Comment:
@gradedSystem I don't think getting an excel version is a good idea. It only adds complexity. Can you check this landing page for the GDP data from WorldBank: https://data.worldbank.org/indicator/NY.GDP.MKTP.CD 

You can find API for CSV format there: https://api.worldbank.org/v2/en/indicator/NY.GDP.MKTP.CD?downloadformat=csv

Reply:
@anuveyatsu I changed the code and with the format=csv